server/service/client.go

```diff
@@ -506,6 +506,17 @@ var errConflictingVPPSetupExperienceDeclarations = errors.New("Couldn't edit app
 // properly plan that separation and refactor so that gitops can be
 // significantly cleaned up and simplified going forward.
 
+func numberWithPluralization(n int, singular string, plural string) string {
+	if n == 1 {
+		return fmt.Sprintf("%d %s", n, singular)
+	}
+	return fmt.Sprintf("%d %s", n, plural)
```

Comment:
```suggestion
    return fmt.Sprintf("%s %s", n, n == 1 ? singular : plural)
```

lol jk golang doesn't believe in ternaries

Reply:
nor does it believe in optional arguments, which would let you leave off the third arg in most cases.

Reply:
ikr